### PR TITLE
updating to use CRLF as newline delimeter in CSV upload.  Also, updat…

### LINKED ISSUE #74: Support for newline character
The current development branch uses newline (`\n`) as an input record delimiter, making embedded newlines  (like those found in Excel when pressing "shift + enter") tricky to handle.

It is proposed to use CRLF (`\r\n`) as a record delimiter instead, thereby enabling embedded newlines in the CSV.

### PR DESCRIPTION
**Pull request by allanbowe**
…ing README to point out the 9.4 only support (now we use proc json).

Closes #74

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Yes. Unprecedented.
 
 #### Server Requirements
 
-- SAS&reg; BI Platform (v9.2 or later)
```
Comment (allanbowe) [inline README.md]: 9.4 with proc json

File: sasautos/h54s.sas [hunk 2]
```diff
@@ -115,7 +115,7 @@
 
   %put h54s ==> === START DESERIALISING TABLE &&_WEBIN_name&baftn. === ;
       data "&&_WEBIN_name&baftn."n;
-        INFILE &&_WEBIN_FILEREF&baftn. LRECL=32767 recfm=v dsd;
```
Comment (allanbowe) [inline sasautos/h54s.sas]: force CRLF as line delimiter

File: src/sasData.js [hunk 3]
```diff
@@ -234,7 +234,7 @@ SasData.prototype.addTable = function(table, macroName, specs) {
       }
     }
     if(i < table.length - 1) {
-      sasCsv += '\n';
```
Comment (allanbowe) [inline src/sasData.js]: adding CRLF on frontend

File: src/sasData.js [hunk 4]
```diff
@@ -161,8 +161,8 @@ SasData.prototype.addTable = function(table, macroName, specs) {
         if(val === true || val === false) {
           throw new h54sError('typeError', 'Boolean value in one of the values (columns) is not allowed');
         }
-        if(type === 'string' && val.indexOf('\n') !== -1) {
-          throw new h54sError('typeError', 'New line character is not supported');
+        if(type === 'string' && val.indexOf('\r\n') !== -1) {
```
Comment (boemska-nik) [inline src/sasData.js]: Let's change this so that \r\n is replaced with a \n, that way all newlines are supported

**Pull request merged**